lib/events/s3sessions/s3handler.go

```diff
@@ -90,17 +101,20 @@ func (s *Config) SetFromURL(in *url.URL, inRegion string) error {
 	if endpoint := in.Query().Get(teleport.Endpoint); endpoint != "" {
 		s.Endpoint = endpoint
 	}
+
+	const boolErrorTemplate = "failed to parse URI %q flag %q - %q, supported values are " +
+		"'1', 't', 'T', 'TRUE', 'true', 'True', '0', 'f', 'F', 'FALSE', 'false', and 'False'."
```

Comment:
Nit: I know it was previously suggested to you here, but I honestly find this message quite confusing. I'm not sure why a user needs to know all forty two options Go can convert to boolean :) TBH I think it was fine with with just true/false even if not technically 100% accurate.

Reply:
Yeah, I definitely didn't mean to suggest we put all the valid options in here, sorry about that!